src/components/GEditor.tsx

```diff
@@ -100,7 +102,7 @@ function GEditor(props: IProps) {
     []
   );
 
-  return <div id={id}/>;
+  return <div id={id}>{children ? children : null}</div>;
```

Comment:
Please make a change like below to simplify the code, also remember to surround the return value with parentheses :)
```jsx
return (
    <div id={id}>
      {children}
    </div>
  );
```

Reply:
Ok, I just pushed these changes.